### PR TITLE
Updated to work with Leaflet 1.4.0

### PR DESCRIPTION
**Pull request by mpuputti**
Leaflet 1.4.0 changed 'form' to 'section'.
This PR updates ActiveLayers to check which one it is, and continues accordingly.

I didn't know which minifyer this project has been using, so I simply made it "manually" in ActiveLayers.min.js.

### CODE REVIEW

**Comment by vogdb** [comment]
Thank you! That was operative. I see that I didn't add CI and test are outdated so I trust you on the changes. Can you please include the next changes:
- delete `bower.json`
- change `"leaflet": "~0.7"` to `"leaflet": "~1.4"` in `package.json`

**Comment by mpuputti** [comment]
@vogdb  Sure, the new patch deletes the `bower.json` and edits the `package.json`.
Please note that I am not familiar with npm or other similar package tools. Instead, I manually copy the js-files into my workspace. So I cannot test the changes this PR makes in the json-files.
Also for your knowledge: I tested my changes with Leaflet versions 1.3.1, 1.4.0 and 1.7.1. So with this PR, your ActiveLayers should continue working with pre-1.4.0 Leaflet. But I have not tested with Leaflet version 0.7, and cannot say anything about compatibility with it.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         }
     ],
     "dependencies": {
-      "leaflet": "~0.7"
+      "leaflet": "~1.4"
```
Comment (vogdb) [inline package.json:17]:
```suggestion
      "leaflet": ">0.7"
```

**Comment by vogdb** [comment]
Thank you for test feedback. I've updated `package.json` accordingly.

**Pull request merged**